e2e/env/env.go

```diff
@@ -125,3 +125,20 @@ func New(config *types.Config, log *zap.SugaredLogger) (*types.Env, error) {
 
 	return env, nil
 }
+
+// GetEnvCluster returns the cluster from the env that matches clusterName.
+// If not found, it returns an empty Cluster and an error.
+func GetEnvCluster(ctx types.Context, clusterName string) (types.Cluster, error) {
```

Comment:
This is the env package so we don't ned to repeat Env in the name.
Also passing the context is not needed, we need only the env.